.pre-commit-config.yaml

```diff
@@ -42,6 +42,16 @@ repos:
     hooks:
       - id: typos
```

Comment:
Maybe remove typos then? I don't think we need two spell checkers.

Reply:
As I said, I don't think one of them is better than another, and I don't see a problem of having both.